python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -213,6 +218,16 @@ def resolve_runs(self, graphene_info: ResolveInfo) -> "Sequence[GrapheneRun]":
         records = self._get_records(graphene_info)
         return [GrapheneRun(record) for record in records]
 
+    def resolve_endTimestamp(self, graphene_info: ResolveInfo) -> Optional[float]:
```

Comment:
@clairelin135  added this field so we wouldn't need to query runs on the client

Reply:
👍 👍